cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -13,205 +13,91 @@ constants:
 phases:
   - name: build
     steps:
-
-      # Get AWS Region
-      - name: AWSRegion
-        action: ExecuteBash
-        inputs:
-          commands:
-            - |
-              set -v
-              echo ${AWS::Region}
-
-      # Get Cookbook name
-      - name: PClusterCookbookVersionName
+      # Get operating system name for variable chaining
+      - name: OperatingSystemName
```

Comment:
Why not combine OperatingSystemName and SystemInfo?

Reply:
Ok got it

Reply:
Why do we set this OS in the image_dna.json when Cookbook could essentially set that for us?
These defaults are used for image_dna.json w